lib/hawkular/alerts/alerts_api.rb

```diff
@@ -559,4 +559,7 @@ def initialize(event_hash)
       super(event_hash)
     end
   end
+
+  AlertsClient = Client
+  deprecate_constant :AlertsClient
```

Comment:
`deprecate_constant` doesn't seem to exist on our base testing version. Will fix it later.